src/witan/app/demo_data.clj

```diff
@@ -3,7 +3,7 @@
             [witan.app.data :as data]
             [witan.app.forecast :as forecast]
             [witan.app.model :as model]
-            [witan.app.user :as user]))
```

Comment:
I guess this is a personal choice, but I prefer to have namespaces expressive (user over u). Is there a compelling reason for using u?